cli/src/maintenance.rs

```diff
@@ -471,11 +476,11 @@ impl SolidoState {
                         lido: self.solido_address,
                         maintainer: self.maintainer_address,
                         validator_vote_account: validator.pubkey,
-                        source_stake_account: validator_stake_account,
+                        source_stake_account: stake_accounts[0].0,
                         destination_unstake_account: validator_unstake_account,
                         stake_authority: self.get_stake_authority(),
                     },
-                    try_unstake_balance,
+                    stake_accounts[0].1.balance.total(),
```

Comment:
We also still need to confirm somewhere that the stake account is fully active, right? Or does it work even when it is activating?

Reply:
It works when it's activating, I added some tests in #396